docs/_tutorials/reference.md

```diff
@@ -98,6 +98,7 @@ Bolt includes a collection of initialization options to customize apps. There ar
 | `unhandledRequestHandler` | Error handler triggered when a request from Slack goes unacknowledged. More details available in the [Error Handling documentation](/bolt-js/concepts#error-handling). |
 | `unhandledRequestTimeoutMillis` | How long to wait, in milliseconds, from the time a request is received to when the `unhandledRequestHandler` should be triggered. Default is `3001`. More details available in the [Error Handling documentation](/bolt-js/concepts#error-handling). |
 | `signatureVerification` | `boolean` that determines whether Bolt should [verify Slack's signature on incoming requests](https://api.slack.com/authentication/verifying-requests-from-slack). Defaults to `true`. |
+| `customPropertiesExtractor` | Optional `function` that can extract custom properties from a request -- for example, extracting custom headers to propagate to other services. More details available in the [Customizing a receiver documentation](/bolt-js/concepts#receiver). |
```

Comment:
Let's clarify two things:
- what the function argument is (at least, what type it is: for HTTP-based receivers, it would be an HTTP request, but for e.g. Socket Mode, it would be a WebSocket message.
- what the function needs to return, and then how those custom properties are available in the context of a wider app. If I understand correctly, the returned object becomes part of the `context` argument provided to Bolt middleware.

Reply:
Indeed, Fil's points can be mentioned quickly in the descrption.

In addition to that, I think we can remove the following part because the linked document section is about how to build your own receiver, not how to customize a built-in one.
> More details available in the [Customizing a receiver documentation](/bolt-js/concepts#receiver).